source/user-manual/reference/ossec-conf/global.rst

```diff
@@ -370,7 +370,7 @@ For example:
 rotate_interval
 ^^^^^^^^^^^^^^^
 
-.. versionadded:: 3.1.0
+.. note:: This option is deprecated since Wazuh 3.9.0. Use the :doc:`logging <./logging>` block instead.
```

Comment:
Why don't we use this syntax?
```
.. deprecated:: 3.9.0
```

Reply:
This would be the result. We need to warn about the successor option.

![imagen](https://user-images.githubusercontent.com/20266121/59516229-2ad5d580-8ec1-11e9-8f8f-0bbb5a7fa864.png)

Reply:
Thanks @crolopez!
I like this one more. Although I would put the note below the table.
Cheers!